Views/Home/Index.cshtml

```diff
@@ -21,3 +21,9 @@
         <p>All your debts are kept safe in your phone.</p>
     </div>
 </div>
+
+<div class='footer'>
+    <label>Get info on new updates</label>
+    <input class='input'> </input>
```

Comment:
Input is a self closing tag. Oh, and by the way, careful with the class name. Use something more self-explanatory.